### ISSUE TITLE
Windows 10: linking bthprops.lib instead of Irprops.lib

### ISSUE THREAD

**Issue by vonBork** [pybluez/pybluez#398]
*PyBluez is not under active development but we are seeking new contributors
to investigate bugs and submit patches.*

## System

 - **Operating System:** Windows
 - **Hardware:** (e.g., Raspberry Pi, external Bluetooth adaptor, etc.) TP-Link USB Bluetooth Adapter
 - **Python Version:** 3.9.4 
 - **PyBluez Version:** 0.23 (installation via pipenv)

## Issue
While installing pybluez via `pipenv install`, pipenv compiles and links windows libraries. One of these libraries is `Irprops.lib`, which can not be linked, as Microsoft removed it from the Windows 10 SDK Build 18890. Microsoft suggests to use` bthprops.lib` instead. Microsoft claims this change would only break Windows XP SP2.

For more informations see https://devblogs.microsoft.com/oldnewthing/20190516-00/?p=102498 and https://blogs.windows.com/windowsdeveloper/2019/05/09/windows-10-sdk-preview-build-18890-available-now/

```
[pipenv.exceptions.InstallError]:     LINK : fatal error LNK1181: Eingabedatei "Irprops.lib" kann nicht ge”ffnet werden.
[pipenv.exceptions.InstallError]:     error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.28.29910\\bin\\HostX86\\x64\\link.exe' failed with exit code 1181
```

Can someone have a look at it?

**Comment by vonBork** [comment]
Thanks in advance.

**Comment by vonBork** [comment]
Ohh I found out, it is already fixed on master, but not in Release 0.23. See https://github.com/pybluez/pybluez/issues/224
Can anybody tell me when Release 0.24 is planed?
